src/applications/vaos/utils/appointment.jsx

```diff
@@ -463,3 +464,126 @@ export function getAppointmentTypeHeader(appt) {
       return appt.purposeOfVisit;
   }
 }
+
+/**
+ * Function to get the appointment duration in minutes. The default is 60 minutes.
+ *
+ * @param {*} appt
+ * @returns
+ */
+function getAppointmentDuration(appt) {
+  const type = getAppointmentType(appt);
+
+  switch (type) {
+    case APPOINTMENT_TYPES.vaAppointment: {
+      const appointmentLength = parseInt(
+        appt.vdsAppointments[0]?.appointmentLength,
+        10,
+      );
+      return isNaN(appointmentLength) ? 60 : appointmentLength;
+    }
+    default:
+      return 60;
+  }
+}
+
+/**
+ * Function to get the appointment address.
+ *
+ * @param {*} appt
+ * @param {*} facility
+ * @returns 'undefined' if there is no address
+ */
+function getAppointmentAddress(appt, facility) {
```

Comment:
I wonder if it makes more sense to say `getAppointmentLocation`? I think location is a better description for something can return "Video conference."

Reply:
The name 'getAppointmentLocation' is taken.